src/cpp/src/continuous_batching/pipeline_base.cpp

```diff
@@ -257,7 +257,8 @@ GenerationHandle
 ContinuousBatchingPipeline::IContinuousBatchingPipeline::add_request(uint64_t request_id,
                                         const std::string& prompt,
                                         const std::vector<ov::Tensor>& rgbs,
-                                        GenerationConfig sampling_params) {
+                                        GenerationConfig sampling_params,
+                                        std::optional<ov::Tensor> token_type_ids) {
```

Comment:
Why do we need to pass `token_type_ids` here if they are overwritten in 272 line anyway?